app/content/drill/[id]/statistics.js

```diff
@@ -32,5 +53,16 @@ export default function Stat() {
   }
   // console.log(drillAttempts);
 
-  return <BarChartScreen drillData={drillAttempts} drillInfo={drillInfo} />;
+  return (
+    <ScrollView
+      // might be awkward to put a scrollview here since barchart component has a scrollview too, but needed a custom
```

Comment:
yeah I just realized that the drill tab stats page calls the same hooks as profile tab stats page. Moved refresh stuff into barchart component. Checked drillData input for drill ID (did) and user ID (uid), since the team tab stats page might have different user ID values than the currently signed in user.